### ISSUE TITLE
Terminate ELK service

### ISSUE THREAD

**Issue by dlapiduz** [18F/cg-agent-q#3]
There is an ELK service in the Docker broker that has been superseded by logs.cloud.gov.

We need to write up a plan to terminate it and communicate with the users about it.

There is relatively a small number of people using this service so it shouldn't be a big deal.

@brittag

**Comment by afeld** [comment]
Is there a not-too-hard way to see the global bindings/tenants/instances?

**Comment by dlapiduz** [comment]
Admin.cloud.gov

On Wednesday, June 1, 2016, Aidan Feldman notifications@github.com wrote:

> Is there a not-too-hard way to see the global bindings/tenants/instances?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/18F/cg-agent-q/issues/3#issuecomment-223156525, or mute
> the thread
> https://github.com/notifications/unsubscribe/AAAR_PF1zB4U8BERxQWE4W2ockqO2nKSks5qHhfHgaJpZM4IryGE
> .

## 

Diego Lapiduz
Director of cloud.gov
diego.lapiduz@gsa.gov
512 766 3434

**Comment by afeld** [comment]
The organizations that have instances:
- [x] `cap` (no binding)
- [ ] `ed` (no binding)
- [ ] `fbopen` - waiting on [shutdown](https://18f.slack.com/archives/fbopen/p1464840928000005)
- [x] `federalist` (no binding)
- [ ] `open-opportunities`

Pinged in the respective channels to see if anyone's using them.

**Comment by afeld** [comment]
As tenants are moved to GovCloud, this problem will solve itself.
